N/ngspice/bundled/patches/drop_ac_malloc.patch

```diff
@@ -0,0 +1,38 @@
+commit 8960318bdde32ba8ea38e268fea8ba130b2ae3ba
+Author: Keno Fischer <keno@juliacomputing.com>
+Date:   Mon Oct 5 03:28:56 2020 -0400
+
+    Drop AC_FUNC_*ALLOC configure macros
```

Comment:
I can't find anything directly referencing these macros in any other builders, but if I remember correctly I also wrestled with these macros before, they're just painful.  I also found other people online just getting rid of them to make things work.

Reply:
packages that use them are supposed to ship the rpl_ versions of these functions, but they basically never do